has-selector.js

```diff
@@ -3,65 +3,81 @@
 (function () {
     "use strict";
 
-    var basis = Date.now(); // Base number used to create unique ids for each alias
-    var outer = [];         // Each entry is a CSS selector corresponding to the outer element
-    var inner = [];         // Each entry is a CSS selector corresponding to the inner element
-    var alias = [];         // Each entry is a unique class name
+    var unique = Date.now(); // Base number used to create unique ids for each alias
+    var outer = [];          // Each entry is a CSS selector corresponding to the outer element
+    var inner = [];          // Each entry is a CSS selector corresponding to the inner element
+    var alias = [];          // Each entry is a unique class name
 
-    addEventListener("mouseenter", toggleAliases);
-    addEventListener("mousedown", toggleAliases);
-    addEventListener("mouseout", toggleAliases);
-    addEventListener("focusin", toggleAliases);
-    addEventListener("change", toggleAliases);
-    addEventListener("click", toggleAliases);
-    addEventListener("input", toggleAliases);
+    parseInlineStylesheets();
+    loadExternalStylesheets();
     addEventListener("blur", toggleAliases);
-    addEventListener("load", toggleAliases);
-    loadStylesheets();
+    addEventListener("input", toggleAliases);
+    addEventListener("click", toggleAliases);
+    addEventListener("change", toggleAliases);
+    addEventListener("focusin", toggleAliases);
+    addEventListener("mouseout", toggleAliases);
+    addEventListener("mousedown", toggleAliases);
+    addEventListener("DOMContentLoaded", toggleAliases);
 
     // This function loads each <link> element's stylesheet so that we can parse it.
-    // Without it, we'd have no way of finding the :has selectors.
-    function loadStylesheets() {
-        var s = "link[rel=stylesheet][href]"; // Selector matching all <link> elements
-        var n = document.querySelectorAll(s); // list of all <link> elements
+    // Without it, we'd have no way of finding the :has selectors within <link> elements.
+    function loadExternalStylesheets() {
+        var s = "link[rel=stylesheet][href]"; // Selector matching <link> elements
```

Comment:
I would love to see longer more descriptive variable names, instead of one letter vars with a comment that describes them.